src/BPSecLib_Public.h

```diff
@@ -186,6 +186,18 @@ typedef struct BSL_BundleRef_s
     void *data; ///< Opaque pointer, not used by the BSL.
 } BSL_BundleRef_t;
 
+typedef enum
+{
+    BSL_REASONCODE_NO_ADDITIONAL_INFO   = 0,
+    BSL_REASONCODE_DEPLETED_STORAGE     = 4,
+    BSL_REASONCODE_BLOCK_UNINTELLIGIBLE = 8,
+    BSL_REASONCODE_MISSING_SECOP        = 12,
+    BSL_REASONCODE_UNKNOWN_SECOP        = 13,
+    BSL_REASONCODE_UNEXPECTED_SECOP     = 14,
+    BSL_REASONCODE_FAILED_SECOP         = 15,
+    BSL_REASONCODE_CONFLICTING_SECOP    = 16
+} BSL_ReasonCode_e;
```

Comment:
Typedefs should be named `_t` for consistency while `enum ..._e {}` can be used also if needed.